### ISSUE TITLE
configuration comment in acceptance.suite.dist.yml

### ISSUE THREAD

**Issue by peterpeter** [joomla-extensions/weblinks#139]
`url: 'http://localhost/tests/joomla-cms3'     # the url that points to the joomla installation at /tests/system/joomla-cms`

....is confusing, since there isn't such a path at this moment, at least before the first run

**Comment by therunnergit** [comment]
You are totally right. The local path now can be configured to whatever you want so the comment is not correct. Note: of course the local path will never exist before you run the test.

@peterpeter can you suggest a better comment?

**Comment by peterpeter** [comment]
I'm not sure. I've just started with weblinks, and I'm still discovering the components used within. But, if I'm not guessing wrong, these path depends on what is set in the RoboFile.dist.ini. If so, a reference to that file would be not bad in that comment. Like 
` the url that points to the joomla installation (see value for cmsPath in RoboFile.dist.ini) `
If I'm wrong, I fear you have to ask someone who knows that ;-)

**Comment by chrisdavenport** [comment]
Suggestion: "The URL that will point to the Joomla installation that the test will create"

Does that sound more accurate? @peterpeter @therunnergit

**Comment by puneet0191** [comment]
"The URL points to Joomla Installation where the test should be executed" @chrisdavenport @zanardigit @peterpeter
